maistra/run-ci.sh

```diff
@@ -15,26 +15,32 @@ export ARCH
 export BUILD_SCM_REVISION="Maistra PR #${PULL_NUMBER:-undefined}"
 export BUILD_SCM_STATUS="SHA=${PULL_PULL_SHA:-undefined}"
 
+COMMON_FLAGS="\
+    --incompatible_linkopts_to_linklibs \
+    --local_ram_resources=12288 \
+    --local_cpu_resources=6 \
+    --jobs=3 \
+    --color=no \
+"
+
+if [ -n "${BAZEL_REMOTE_CACHE}" ]; then
+  COMMON_FLAGS+=" --remote_cache=${BAZEL_REMOTE_CACHE} "
+elif [ -n "${BAZEL_DISK_CACHE}" ]; then
+  COMMON_FLAGS+=" --disk_cache=${BAZEL_DISK_CACHE} "
+fi
```

Comment:
Should there be an `else` clause here so the `--disk_cache=/bazel-cache` option still gets used as the default if neither of the environment variables are set?

Reply:
I was just thinking about folks who build in a docker container with a `/bazel-cache' mount, and expect it to just work. With this change, the default behaviour will change to using ~/.cache/bazel instead. However, I guess it's simple enough to set the additional environment variable.

Reply:
yeah, that's what I'm doing